### PR TITLE
Explore: Add orangeDark color to theme

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Adds orangeDark color to the theme as this color is used by multiple buttons in Explore. Also, it refactors the code that used this color to get it from theme, rather than declaring it locally.

### CODE REVIEW

**Comment by dprokop** [comment]
Nit: I think we should update `grafana-ui/src/themes/_variables.dark.scss.tmpl.ts` and `grafana-ui/src/themes/_variables.light.scss.tmpl.ts` too to make this colour available in SASS as well

**Review by dprokop — APPROVED** [review]

File: public/app/features/explore/LiveTailButton.tsx [hunk 1]
```diff
@@ -5,14 +5,13 @@ import memoizeOne from 'memoize-one';
 import tinycolor from 'tinycolor2';
 import { CSSTransition } from 'react-transition-group';
 
-import { GrafanaTheme, GrafanaThemeType, useTheme } from '@grafana/ui';
+import { GrafanaTheme, useTheme } from '@grafana/ui';
 
 const getStyles = memoizeOne((theme: GrafanaTheme) => {
-  const orange = theme.type === GrafanaThemeType.Dark ? '#FF780A' : '#ED5700';
-  const orangeLighter = tinycolor(orange)
+  const orangeLighter = tinycolor(theme.colors.orangeDark)
```
Comment (dprokop) [inline public/app/features/explore/LiveTailButton.tsx]: Isn't this one of the colours Jess defined in new oranges (ref https://github.com/grafana/grafana/pull/19274#discussion_r327526964)? Same with `pulseTextColor` below. If yes, then I suggest introducing these colours to the theme.

Comment (ivanahuckova) [reply public/app/features/explore/LiveTailButton.tsx]: No it is not, when I console-logged the colours, they were:
#ff943d for orangeLighter
#918378 for pulseTextColor

**Comment by ivanahuckova** [comment]
> Nit: I think we should update `grafana-ui/src/themes/_variables.dark.scss.tmpl.ts` and `grafana-ui/src/themes/_variables.light.scss.tmpl.ts` too to make this colour available in SASS as well

@dprokop  Great nit pick! 🙂I've updated the documents so the colours are available in SASS as well.

**Comment by dprokop** [comment]
@ivanahuckova one more thing. When you modified theme template files, you probably didn't have `yarn start` running, as I cannot see `*.generated.css` files in this PR. Plese see docs around it here: https://github.com/grafana/grafana/blob/master/style_guides/themes.md#how-can-i-modify-sass-variable-files

**Pull request merged**